docs/development/translations.rst

```diff
@@ -53,7 +53,15 @@ In Templates
 
 Inside of Jinja2 templates the standard
 `Jinja2 i18n extension <http://jinja.pocoo.org/docs/dev/extensions/#newstyle-gettext>`_
-configured with ``newstyle=True``.
+has been configured with ``newstyle=True``.
+
+You can use it like so:
+
+.. code:: python
```

Comment:
This isn't python code, it should probably either be html, nothing, or jinja2 if Sphinx has that.